db/migrate/20170827091406_change_container_quota_items_columns_to_float.rb

```diff
@@ -0,0 +1,36 @@
+class ChangeContainerQuotaItemsColumnsToFloat < ActiveRecord::Migration[5.0]
+  class ContainerQuotaItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_quota_items, :quota_desired_float, :float
+    add_column :container_quota_items, :quota_enforced_float, :float
+    add_column :container_quota_items, :quota_observed_float, :float
+
+    ContainerQuotaItem.all.each do |item|
```

Comment:
can you use `find_each` to batch this instead of loading `all` at once?
otherwise, LGTM.

Reply:
done :+1: